### DISCUSSION TITLE
Cypress test running locally but skipping tests on github actions beforeEach & afterEach Hook

### DISCUSSION THREAD

**Discussion by EashBhanu-CCD** [cypress-io/cypress#27692] [CI setup]
I am trying to visit url in beforeEach block but everttime it fails in github action

This is github workflow file :

name: Run Cypress Tests Automation

on:
  push:
    branches:
      - feature/automation-testing  

jobs:
  run-cypress-tests:
    runs-on: ubuntu-latest  

    steps:
      - name: Checkout repo
        uses: actions/checkout@v2
        with:
          repository: RepoName
          token: ${{ secrets.MYPAT }}

      # - name: Set up Node.js
      #   uses: actions/setup-node@v2
      #   with:
      #     node-version: '18'  

      # - name: Install dependencies
      #   run: |
      #     npm install

      # - name: Install Cypress
      #   run: |
      #     npm install cypress --save-dev

      # - name: Run Cypress Tests
      #   run: |
      #     npm run test

      - name: Cypress run
        uses: cypress-io/github-action@v5
        with:
          start: npm install
          build: npm run test
          browser: chrome

          

This is the error am getting in github action : 

Customer Page - Partner
1) "before each" hook for "Click on customer icon & verify headers"

0 passing (19s)
1 failing

Customer Page - Partner
"before each" hook for "Click on customer icon & verify headers":
AssertionError: Timed out retrying after 4000ms: expected '/auth' to include '/user/'

**Comment by byKosta** [comment]
Hello @EashBhanu-CCD Have you tried using? Maybe this will help you find problem https://github.com/nektos/act
I'm not sure if it's still relevant or not. If not, please close it.
